airbyte-webapp/src/components/connection/CatalogTree/next/StreamDetailsPanel.module.scss

```diff
@@ -12,8 +12,13 @@
   left: variables.$width-size-menu;
   z-index: 1000;
   width: calc(100% - variables.$width-size-menu);
-  height: calc(100vh - 100px);
+  height: 87vh;
```

Comment:
Just curious about **height: 87vh;**. Why exactly 87vh? :grin:

Reply:
Good question! 😄 
I didn't find any design when the Details panel is opened, hence I don't know what the height of the gap should be. 
So I set `87vh` as the approximate value
@edmundito do you happen to know where I can find it?

<img width="470" alt="image" src="https://user-images.githubusercontent.com/20929344/204806220-df132ea5-0519-4260-8d49-24a9e4f954cf.png">

Reply:
Fixed, now gap on top is `100vh - 100px`